modules/src/ics02_client/client_def.rs

```diff
@@ -18,6 +18,12 @@ use crate::Height;
 
 use tendermint_proto::{DomainType, Error, Kind};
 
+pub const TENDERMINT_CLIENT_STATE_TYPE_URL: &str = "/ibc.lightclients.tendermint.v1.ClientState";
+pub const TENDERMINT_CONSENSUS_STATE_TYPE_URL: &str =
+    "/ibc.lightclients.tendermint.v1.ConsensusState";
+pub const MOCK_CLIENT_STATE_TYPE_URL: &str = "/ibc.mock.ClientState";
+pub const MOCK_CONSENSUS_STATE_TYPE_URL: &str = "/ibc.mock.ConsensusState";
```

Comment:
Probably not the ideal way to define these, let me know what you think. Maybe we leave this as is and address it in #254?